### PR TITLE
[Pulsar-Flink] Add Scala Examples

### PR DESCRIPTION
**Pull request by erenavsarogullari**
### Motivation
This PR aims to help Scala community by adding Scala Examples on Flink Connector.

### Modifications
1- `FlinkPulsarBatchSinkScalaExample` is added for `PulsarOutputFormat`
2- `FlinkPulsarBatchCsvSinkScalaExample` is added for `PulsarCsvOutputFormat`
3- `FlinkPulsarBatchJsonSinkScalaExample` is added for `PulsarJsonOutputFormat`
4- Extra comments are added to `FlinkPulsarBatchSinkExample`

### CODE REVIEW

File: pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.apache.flink.batch.connectors.pulsar.example
```
Comment (merlimat) [inline pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala]: Missing ASF header

Comment (erenavsarogullari) [reply pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala]: Thanks, addressed ;)

**Comment by merlimat** [comment]
@erenavsarogullari It looks like the license headers do not match exactly the pattern and the checker is rejecting them: 

```
2018-11-27\T\23:38:25.874 [INFO] Checking licenses...
2018-11-27\T\23:38:25.899 [WARNING] Missing header in: /home/jenkins/jenkins-slave/workspace/pulsar_precommit_java8/pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala
2018-11-27\T\23:38:25.899 [WARNING] Missing header in: /home/jenkins/jenkins-slave/workspace/pulsar_precommit_java8/pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchCsvSinkScalaExample.scala
2018-11-27\T\23:38:25.899 [WARNING] Missing header in: /home/jenkins/jenkins-slave/workspace/pulsar_precommit_java8/pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchSinkScalaExample.scala
2018-11-27\T\23:38:25.903 [INFO] ------------------------------------------------------------------------

```

It should get fixed with `mvn license:format`

**Comment by erenavsarogullari** [comment]
@merlimat Thanks. Yes, extra whitespace problem has occurred on license. Patch submitted

**Pull request merged**